apidef/oas/operation.go

```diff
@@ -116,6 +120,7 @@ func (s *OAS) fillPathsAndOperations(ep apidef.ExtendedPathsSet) {
 	s.fillAllowance(ep.Ignored, ignoreAuthentication)
 	s.fillTransformRequestMethod(ep.MethodTransforms)
 	s.fillTransformRequestBody(ep.Transform)
```

Comment:
```suggestion
	s.fillTransformRequestBody(ep.TransformRequest)
```

This is what I was expecting here, naming?